extensions/integration-tests/src/schemaCompare.test.ts

```diff
@@ -237,8 +237,8 @@ class SchemaCompareTester {
 					}
 				});
 			}
+			// TODO: add proper validation for task completion to ensure all tasks successfully complete before exiting test
```

Comment:
Make an issue for this and link it as well please

Reply:
Created issue #6230